src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -32,23 +29,19 @@ public LineService(LineRepository lineRepository, StationService stationService)
     public LineResponse saveLine(LineRequest request) {
         Station upStation = stationService.findById(request.getUpStationId());
         Station downStation = stationService.findById(request.getDownStationId());
-        Line persistLine = lineRepository.save(new Line(request.getName(), request.getColor(), upStation, downStation, request.getDistance()));
-        List<StationResponse> stations = getStations(persistLine).stream()
-                .map(it -> StationResponse.of(it))
-                .collect(Collectors.toList());
-        return LineResponse.of(persistLine, stations);
+        Line persistLine = lineRepository.save(
+            new Line(request.getName(), request.getColor(), upStation, downStation,
+                new Distance(request.getDistance())));
+
+        return LineResponse.of(persistLine);
     }
 
     public List<LineResponse> findLines() {
```

Comment:
```suggestion
    @Transactional(readOnly = true)
    public List<LineResponse> findLines() {
```

조회만 하는 기능의 경우 `readOnly` 옵션을 사용해볼 수 있겠네요 :)